client/my-sites/stats/post-trends/style.scss

```diff
@@ -163,55 +136,106 @@
 	}
 
 	&.is-hovered {
-		border-color: var(--color-primary);
+		transform: scale(1.2);
 	}
 
 	.is-loading & {
 		animation: loading-fade 1.6s ease-in-out infinite;
 	}
 }
 
-@include breakpoint-deprecated( ">1040px" ) {
-	.post-trends__scroll-left,
-	.post-trends__scroll-right {
-		display: none;
-	}
-}
-
 .post-trends__key-container {
-	@include clear-fix;
-	padding-top: 10px;
-	float: right;
-
-	@include breakpoint-deprecated( "<960px" ) {
-		float: none;
-		margin: 0 auto;
-		margin-bottom: 25px;
-		text-align: center;
-	}
-}
-
-.post-trends__key-label,
-.post-trends__key {
-	display: inline-block;
+	margin-top: 16px;
+	margin-bottom: 48px;
+	float: none;
+	display: flex;
+	align-items: center;
+	justify-content: center;
 }
 
 .post-trends__key-label {
-	font-size: $font-body-extra-small;
-	color: var(--color-neutral);
-	letter-spacing: 0.1em;
-	text-transform: uppercase;
+	font-weight: 400;
+	font-size: $font-body-small;
+	line-height: 18px;
+	color: var(--color-text);
+	text-transform: none;
+	letter-spacing: normal;
 }
 
 .post-trends__key {
-	margin: 0;
+	border-radius: $common-border-radius;
+	overflow: hidden;
+	padding: 0;
+	margin: 0 16px;
 	list-style-type: none;
-	padding: 2px 9px 0 5px;
+	display: flex;
 
 	.post-trends__key-day {
-		width: 10px;
-		height: 10px;
-		float: left;
-		margin-left: 3px;
+		width: 25px;
+		height: 25px;
+		margin: 0;
+	}
+}
+
+.post-trends-year-navigation {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+
+	.post-trends-year-navigation__previous,
+	.post-trends-year-navigation__next {
+		height: 24px;
+
+		&:focus,
+		&:active {
+			outline: thin dotted;
+		}
+	}
+
+	@media (min-width: 661px) {
+		padding-right: 0;
+	}
+}
+
+
+.tooltip.popover.post-trends-day__tooltip {
+	&.is-top .popover__arrow {
+		border-top-color: var(--color-neutral-100);
+		box-shadow: 0 1px 2px $tooltip-box-shadow-color;
+	}
+
+	.popover__inner {
+		background-color: var(--color-neutral-100);
+		box-shadow: 0 1px 2px $tooltip-box-shadow-color;
+		border-radius: 4px;
+		padding: 16px 24px;
```

Comment:
Do we want to migrate some of these styles into the tooltip component in a follow-up PR?

https://github.com/Automattic/wp-calypso/blob/cffafad574ca980b9fd0483b3101ae326dae1bda/client/components/tooltip/style.scss#L177

Reply:
Sure! This makes total sense to me! I'll create a follow-up PR with this.

Reply:
This is addressed on this follow-up PR: https://github.com/Automattic/wp-calypso/pull/71676